saleor/graphql/translations/mutations.py

```diff
@@ -85,10 +105,16 @@ def clean_node_id(cls, **data):
 
         return node_id, model_type
 
+    @classmethod
+    def validate_input(cls, input_data):
+        validate_input(cls._meta.model, input_data)
```

Comment:
It looks weird. I would rename this function because my first thought was: recursion.